services/masterdata/src/lib/components/masterdata-type-edit-form/masterdata-type-edit-form.component.html

```diff
@@ -131,7 +136,11 @@
 			<div class="form-actions">
 				<div>
 					<lens-button
-						label="{{ saveBtnText }}"
+						label="{{
```

Comment:
suggestion: move to typescript to keep html clean

Reply:
@Johanh-od I need more details?

Reply:
Good point! I've updated the code.